test/unit/aggregate_store_test.rb

```diff
@@ -18,13 +20,13 @@ def add(attr, message)
 
   context "aggregate_attribute" do
     setup do
-      @store = Class.new {}
+      @store = Class.new { }
```

Comment:
Shouldn't it be `{}`?

Reply:
Same question for the other instances of this as well

Reply:
That was a rubocop autocorrect change....  I am inclined to not fight it.

Reply:
Actually, no, this is an empty block being passed, not an empty hash, so the spaces are right.